src/toil/jobStores/abstractJobStore.py

```diff
@@ -114,6 +158,81 @@ def _checkJobStoreCreation(create, exists, jobStoreString):
             raise JobStoreCreationException("The job store '%s' does not exist, so there "
                                             "is nothing to restart." % jobStoreString)
 
+    def importFile(self, srcUrl):
+        """
+        Imports the file pointed at by sourceUrl into jobStore. The jobStoreFileId of the new
```

Comment:
Just realized I forgot to s/jobStore/job store/ ... sorry about that,  I'll get it on the next round.